### ISSUE TITLE
Missing config.json

### ISSUE THREAD

**Issue by kid-icarus** [meandavejustice/metastaseis#26]
Following install instructions, at `npm start` I get

`Error: Cannot find module './config.json'`

Looks like this was left out, wanna make a config.json-dist or something like that with some sensible defaults and then include instructions to copy to config.json?

**Comment by meandavejustice** [comment]
Only place `config.json` is being used atm is https://github.com/meandavejustice/metastaseis/blob/master/server.js#L13

should add this to readme, but for now if u wanna run this just do:

``` json
{
"user": "ausername@gmail.com",
"password": "getRekt420"
}
```
for the config.json

**Comment by meandavejustice** [comment]
actually I should just remove that feedback page all together and just add a link to my email

**Comment by kid-icarus** [comment]
Fixed d365b0a
